### PR TITLE
feat: combine konvoy and kommander into one pr

### PR DESCRIPTION
**Pull request by mstruebing**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-77810

## Description of changes being made

This combines kommander2/konvoy2 PRs into one sync PR.
I left the single ones alive as a fallback for now, until we have tested this a bit.

https://mesosphere.slack.com/archives/C02AVBS638R/p1628286710018400

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: ci/pull-upstreams.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```
Comment (vacas5) [inline ci/pull-upstreams.sh:1]: what's different now that this file needs to be committed?

**Review by vacas5 — COMMENTED** [review]
I may need a bit to wrap my head around this. I understand what's happening in the cron job. Does this effectively accomplish having a single PR that we could potentially proxy behind dev-docs.d2iq.com as mentioned?

File: ci/pull-upstreams.sh [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+# Will be manipulated by construct_branch_name and construct_pr_name
+PR_NAME="Sync"
+BRANCH_NAME="autosync/"
+
+# Renamed for better readability
+ARGC="$#"
+
+construct_branch_name "$@"
```
Comment (vacas5) [inline ci/pull-upstreams.sh:86]: I wonder if we should simply create a long lived branch. I guess the benefit here is that we don't have to keep rebasing it with changes on main.

File: ci/pull-upstreams.sh (hunk 1 above)
Comment (mstruebing) [reply ci/pull-upstreams.sh:1]: This file can pull more than one other repo into one PR while the other only is able to pull one.

**Comment by mstruebing** [comment]
> Does this effectively accomplish having a single PR that we could potentially proxy behind dev-docs.d2iq.com as mentioned?

It has a single branch, but I think the intention here (at least from my side) was to be easier to be reviewed from docs people.
We may use it also for proxying, but I didn't thought about it yet as I wasn't aware that this is a combined requirement.

**Review by vacas5 — APPROVED** [review]
I think this is an improvement! thanks @mstruebing

**Review by takirala — APPROVED** [review]
:tada:

**Pull request merged**